src/itemstackmetadata.h

```diff
@@ -20,13 +20,35 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #pragma once
 
 #include "metadata.h"
+#include "tool.h"
 
 class Inventory;
 class IItemDefManager;
 
 class ItemStackMetadata : public Metadata
 {
 public:
+	ItemStackMetadata() : toolcaps_overridden(false) {}
+
+	// Overriddes
+	virtual void clear() override;
```

Comment:
no need for `virtual` with `override`. (`virtual` is only needed if this class was subclassed.) So just use `override`

see https://stackoverflow.com/a/39932616

Reply:
Done